sources/packages/api/src-sql/DeleteCascades/Drop-cascade-rules.sql

```diff
@@ -0,0 +1,339 @@
+
+-- Delete reference between disbursement_schedules and student_assessments. --
+ALTER TABLE
+    sims.disbursement_schedules DROP CONSTRAINT IF EXISTS disbursement_schedules_student_assessment_id_fkey;
+
+-- Create reference between disbursement_schedules and student_assessments without delete cascade. --
+ALTER TABLE
+    sims.disbursement_schedules
+ADD
+    CONSTRAINT disbursement_schedules_student_assessment_id_fkey FOREIGN KEY(student_assessment_id) REFERENCES sims.student_assessments(id);
+
+-- Delete reference between disbursement_values and disbursement_schedules --
+ALTER TABLE
+    sims.disbursement_values DROP CONSTRAINT IF EXISTS disbursement_values_disbursement_schedule_id_fkey;
+
+-- Create reference between disbursement_values and disbursement_schedules without delete cascade. --
+ALTER TABLE
+    sims.disbursement_values
+ADD
+    CONSTRAINT disbursement_values_disbursement_schedule_id_fkey FOREIGN KEY(disbursement_schedule_id) REFERENCES sims.disbursement_schedules(id);
+
+-- Delete reference between education_programs and institutions. --
+ALTER TABLE
+    sims.education_programs DROP CONSTRAINT IF EXISTS education_programs_institution_id_fkey;
+
+-- Create reference between education_programs and institutions without delete cascade.--
+ALTER TABLE
+    sims.education_programs
+ADD
+    CONSTRAINT education_programs_institution_id_fkey FOREIGN KEY(institution_id) REFERENCES sims.institutions(id);
+
+-- Delete reference between education_programs_offerings and education_programs. --
+ALTER TABLE
+    sims.education_programs_offerings DROP CONSTRAINT IF EXISTS education_programs_offerings_program_id_fkey;
+
+-- Create reference between education_programs_offerings and education_programs without delete cascade.--
+ALTER TABLE
+    sims.education_programs_offerings
+ADD
+    CONSTRAINT education_programs_offerings_program_id_fkey FOREIGN KEY(program_id) REFERENCES sims.education_programs(id);
+
+-- Delete reference between education_programs_offerings and institution_locations. --
+ALTER TABLE
+    sims.education_programs_offerings DROP CONSTRAINT IF EXISTS education_programs_offerings_location_id_fkey;
+
+-- Create reference between education_programs_offerings and institution_locations without delete cascade. --
+ALTER TABLE
+    sims.education_programs_offerings
+ADD
+    CONSTRAINT education_programs_offerings_location_id_fkey FOREIGN KEY(location_id) REFERENCES sims.institution_locations(id);
+
+-- Delete reference between institution_users and users. --
+ALTER TABLE
+    sims.institution_users DROP CONSTRAINT IF EXISTS fk_institution_user_id_202105180001;
+
+-- Create reference between institution_users and users without delete cascade.--
+ALTER TABLE
+    sims.institution_users
+ADD
+    CONSTRAINT fk_institution_user_id_202105180001 FOREIGN KEY(user_id) REFERENCES sims.users(id);
```

Comment:
While creating the new contraint, can we follow the convention from the automatically generated ones that apparently are the majority?